### PR TITLE
docs: update admin dashboard quickstart

### PR DESCRIPTION
**Pull request by shahednasser**
Updated the admin dashboard quickstart for the release of v1.8 and the Admin Plugin.

Closes DOCS-652

### CODE REVIEW

File: docs/content/admin/quickstart.mdx [hunk 1]
```diff
@@ -33,79 +33,146 @@ You can install Node from the [official website](https://nodejs.org/en/).
...
+```bash npm2yarn
+npm install @medusajs/admin
+```
 
-## Install the Admin
+### Step 2: Add Admin to Medusa Configurations
+
+In `medusa-config.js`, add the admin plugin into the array of `plugins`:
+
+```js title=medusa-config.js
```
Comment (kasperkristensen) [inline docs/content/admin/quickstart.mdx:50]: note: If a user intends to deploy the admin separately (e.g. Vercel), they should not add anything to their `medusa-config`.

File: docs/content/admin/quickstart.mdx [hunk 2]
```diff
@@ -33,79 +33,146 @@ You can install Node from the [official website](https://nodejs.org/en/).
...
+### Step 1: Install the Package
 
-<a href="https://app.netlify.com/start/deploy?repository=https://github.com/medusajs/admin" className="img-url">
-  <img src="https://www.netlify.com/img/deploy/button.svg" alt="Deploy to Netlify" className="no-zoom-img" />
-</a>
+In the directory of your Medusa backend, run the following command to install admin dashboard:
 
----
+```bash npm2yarn
+npm install @medusajs/admin
```
Comment (kasperkristensen) [inline docs/content/admin/quickstart.mdx:43]: note: if the user intends to deploy the admin separately, they can add the plugin as a `devDependency` instead, as they should only be using the build tool aspect and not inject any code that will be used at runtime.

File: docs/content/admin/quickstart.mdx [hunk 3]
```diff
@@ -33,79 +33,146 @@ You can install Node from the [official website](https://nodejs.org/en/).
...
    negativeQuestion="Please describe the issue you faced."
 />
 
-:::tip
-
-If you run into errors during the installation, check out [this troubleshooting guide](../troubleshooting/common-installation-errors.mdx).
+---
 
-:::
+## Serve the Admin Dashboard Separately
```
Comment (kasperkristensen) [inline docs/content/admin/quickstart.mdx:126]: Would it make sense to change this section, so it simply links to the deployment to Vercel docs, and note that they can use `medusa-admin dev` to spin up the admin in development?

File: docs/content/admin/quickstart.mdx [hunk 4]
```diff
@@ -33,79 +33,146 @@ You can install Node from the [official website](https://nodejs.org/en/).
...
 
----
+If you have the `serve` option set to `false`, or if you need to serve the admin dashboard separately from the Medusa backend, add the following script to `package.json`:
 
-## Test it Out
+```json title=package.json
+{
+  "scripts": {
+    // other scripts...
+    "serve:admin": "medusa-admin dev"
```
Comment (kasperkristensen) [inline docs/content/admin/quickstart.mdx:134]: I am a bit afraid that the wording could result in people assuming that they can set up their server, so it runs this command to serve the admin in production. The command spins up a development build of the admin, which would result in their server crashing as Vite uses `fs.watchFile` under the hood on each file. 

```suggestion
    "dev:admin": "medusa-admin dev"
```

File: docs/content/admin/quickstart.mdx [hunk 5]
```diff
@@ -33,79 +33,146 @@ You can install Node from the [official website](https://nodejs.org/en/).
...
 
-![Admin Log In](https://res.cloudinary.com/dza7lstvk/image/upload/v1668001604/Medusa%20Docs/Screenshots/XYqMCo9_hq1fsv.png)
+You can change the default Medusa backend URL (`http://localhost:9000`) that the admin uses to interact with the backend's APIs by passing a `--backend` or `-b` option to the command:
 
-Use your Medusa admin’s user credentials to log in.
+```json title=package.json
+{
+  "scripts": {
+    // other scripts...
+    "serve:admin": "medusa-admin dev --backend example.com"
```
Comment (kasperkristensen) [inline docs/content/admin/quickstart.mdx:168]:
```suggestion
    "dev:admin": "medusa-admin dev --backend example.com"
```

**Review by kasperkristensen — COMMENTED** [review]
Looks good, I've added a few comments 💪

**Comment by shahednasser** [comment]
Thanks for the review @kasperkristensen ! I think your comments shed a light on a different way this documentation should be presented to make it more clear how the admin works and the different ways it can be used, so I've changed it to provide two different approaches of installing the admin

**Pull request merged**